src/Compilers/VisualBasic/Portable/VBResources.Designer.vb

```diff
@@ -9010,6 +9010,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
         
         '''<summary>
+        '''  Looks up a localized string similar to An error occurred while writing the output file: {0}.
+        '''</summary>
+        Friend ReadOnly Property ERR_PeWritingFailure() As String
+            Get
+                Return ResourceManager.GetString("ERR_PeWritingFailure", resourceCulture)
+            End Get
+        End Property
```

Comment:
Hmm, a bit surprised this was needed.  How was this code compiling before?  

Want to make sure we're not missing a case in our update 2 bits here. 

@TyOverby